behind-the-veil-siteroot/imports/ui/components/tabs/Tabs.jsx

```diff
@@ -14,15 +15,15 @@ import { Tab } from '@headlessui/react'
  * @param {Array} tabPanels Text/components to be rendered in the tab panel. Each tab panel should correspond to the tab of the same index in the accompanying tabs array
  * @returns tabs component
  */
-const Tabs = ({ tabs = [], tabPanels = [] }) => {
+const Tabs = ({ tabs = [], tabPanels = [], tabsClassName, tabPanelsClassName}) => {
     return (
         <Tab.Group>
-            <Tab.List className="text-dark-grey main-text border-b-2 border-light-grey">
+            <Tab.List className={classNames("text-dark-grey main-text border-b-2 border-light-grey", tabsClassName)}>
```

Comment:
Can you add whatever tailwind's syntax is for overflow: auto here in the base tab list class name? I forgot to do that when creating this component. Resizing results in this, and i think we want it to be scrollable instead of overflowing:
![Screenshot 2024-05-07 at 6 35 28 pm](https://github.com/Monash-FIT3170/Behind-The-Veil/assets/95511987/2b1009ee-0ebd-4926-981b-f49fb38853ee)

Reply:
overflow-auto added to Tab component. Now scrollable instead of overflowing.

Issue should be resolved.